packages/eyeglass/@types/node-sass/index.d.ts

```diff
@@ -0,0 +1,65 @@
+// Type definitions for Node Sass v3.10.1

```

Comment:
Typically, I have not been committing the type definitions to git. Instead I have been treating them as build artifacts shipped only as packages.

One downside is that for folks who are pointing at a git-ref rather then a release NPM package, they must first manually build these typings.

I'm not sure which approach is right, so I'm curious what your thoughts as.

Reply:
I have the same policy but I had to change the type definition because it was wrong. the system that is used by DefinitelyTyped is actively hostile to linking to an unbuilt repo. My goal is to combine the types in SassImplementation with the main node-sass types and to reconcile this against the type definitions generated for dart-sass. I would like to do this as a follow-on project. I have filed https://github.com/sass-eyeglass/eyeglass/issues/198 to track this.